docs/src/auto_examples/tutorials/run_annoy.py

```diff
@@ -138,7 +137,7 @@ def avg_query_time(annoy_index=None, queries=1000):
         total_time += time.process_time() - start_time
     return total_time / queries
 
-queries = 10000
+queries = 1000
```

Comment:
@mpenkov drawing your attention to this. Most other changes in this PR are cosmetic / documentation, but here I changed the actual value.

The idea was to speed up the tests. I saw no benefit to averaging over 10000 samples compared to 1000 samples.